Source/DafnyLanguageServer/Program.cs

```diff
@@ -60,4 +62,15 @@ public class Program {
         .AddSerilog(Log.Logger);
     }
   }
+  internal class LogWriter : TextWriter {
+    public override void Write(char value) {
+      Log.Write(LogEventLevel.Verbose, value.ToString());
+    }
+
+    public override void Write(string? value) {
+      Log.Write(LogEventLevel.Verbose, value);
+    }
```

Comment:
Two suggestions:
- Use `Log.Logger` with the suitable method for the desired log-level.
- Always use a format string to not accidentally provide format-patterns to the logging framework which can cause errors (or even worse). Furthermore, it makes it easier to scan the logs for this kind of log messages.


```suggestion
    public override void Write(char value) {
      Log.Logger.Verbose("unexpected console output: {value}", value);
    }

    public override void Write(string? value) {
      Log.Logger.Verbose("unexpected console output: {value}", value);
    }
```

PS: You may want to test my suggestions, I'm not entirely sure that I got it right.